### ISSUE TITLE
interations = untilConvergence

### ISSUE THREAD

**Issue by florianhartig** [florianhartig/BayesianTools#27]
I got a few comments that it would be nice to set iterations until convergence in runMCMC.

I guess though that this would require an outer wrapper in runMCMC to stop, check convergence, and restart for n chains. 

What do you guys think?

**Comment by woodwards** [comment]
Yes I would like this, particularly it should work where there is burnin.
